packages/dev/loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -2012,7 +2012,14 @@ export class GLTFLoader implements IGLTFLoader {
 
         const engine = this._babylonScene.getEngine();
 
-        if (accessor.sparse || accessor.bufferView == undefined) {
+        if (accessor.sparse) {

```

Comment:
This shouldn't be reverted.

```suggestion
        if (accessor.sparse || accessor.bufferView == undefined) {
```